features/user_sessions.feature

```diff
@@ -46,3 +46,66 @@ Feature: User sessions
     When I sign out
     And I go to the usagov's Dashboard page
     Then I should see "Security Notification"
+  
+  @javascript
```

Comment:
Use the `@javascript` tag sparingly, as it uses a slower driver. The only scenario that should need it is the one testing the dropdown functionality in the header.